logger.js

```diff
@@ -51,6 +51,16 @@ const formatter = printf((info) => {
   return JSON.stringify(logObject);
 });
 
+// API workaround for creating log directory in Vercel
+import axios from "axios";
+
+const logDirCreationEndpoint =
+  "https://digital-collections-git-feature-dr-2715logging-nypl.vercel.app/api/createLogDir";
```

Comment:
This should work if it's `/api/createLogDir`.

Reply:
As in `const logDirCreationEndpoint = '/api/createLogDir'` ?